app/models/planet.py

```diff
@@ -0,0 +1,24 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    diameter= db.Column(db.Float)
```

Comment:
When designing your class, think about whether or not each attribute should be required. I can see a case for name being a required attribute!